components/Layout/Layout.tsx

```diff
@@ -69,7 +72,11 @@ const Layout = ({ children, isHome, filters, handleFilters }: ILayoutProps) => {
         />
       </div>
 
-      <div className={`z-20 lg:block ${isOpen ? "block" : "hidden"}`}>
+      <div
+        className={`fixed bottom-0 right-0 z-20 transform pb-4 pr-4 transition duration-300 lg:translate-y-0 ${
+          isOpen ? "" : "translate-y-full"
```

Comment:
and `isOpen ? "" : "translate-y-full"` to `!isOpen && "translate-y-full"`

Reply:
done ;)